src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -17,8 +22,19 @@ export class DashboardContainer extends Component {
   state = {
     user: {},
     logPoints: false,
+    filteredUserActivities: null,
+    filterBy: [
+      { name: 'Select All', checked: false },
```

Comment:
How come the `Select All` is properly cased but the other items are not?

Reply:
Ahh, 😄